.claude/scripts/cluster-provision.sh

```
@@ -0,0 +1,350 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Provision an ephemeral OpenShift cluster via the OpenShift CI Gangway API.
+# Claims a cluster from a Hive ClusterPool, downloads kubeconfig, validates.
+#
+# Usage: cluster-provision.sh [up|down|status] [KUBECONFIG_PATH] [OCP_VERSION]
+
+ACTION="${1:-status}"
+KUBECONFIG_OUT="${2:-/tmp/k}"
+OCP_VERSION="${3:-4.18}"
```

Shellcheck (0.11.0)

[warning] OCP_VERSION appears unused. Verify use (or export if used externally).

(SC2034)